scripts/tests/panic_on_invalid_genesis.sh

```diff
@@ -2,8 +2,8 @@
 set -ex
 # Setup env
 export TORII_P2P_ADDR='127.0.0.1:1341'
-export TORII_API_URL='127.0.0.1:8084'
-export TORII_TELEMETRY_URL='127.0.0.1:8184'
+export TORII_API_ADDR='127.0.0.1:8084'
```

Comment:
Please rename back to `URL`. I know that there's a subtle difference, but it's not worth the headache during upgrade.